docs/environment_variables.rst

```diff
@@ -0,0 +1,34 @@
+.. meta::
+   :description: HSA runtime implementation
+   :keywords: ROCR, ROCm, library, tool, runtime
+
+.. _environment-variables:
+
+Environment variables
+========================
+
+The following table lists the useful environment variables.
+
+.. list-table:: ROCR environment variables
+    :header-rows: 1
+    
+    * - Environment Variable
+      - Possible values
+      - Description
+
+    * - HSA_ENABLE_SDMA
+      - 
+        * 0: Disabled
+        * 1: Enabled (default)
+      - This enables or disables the use of DMA engines in all copy directions (Host-to-Device, Device-to-Host, Device-to-Device) when using the
```

Comment:
```suggestion
      - Specifies the use of DMA engines in all copy directions (Host-to-Device, Device-to-Host, Device-to-Device) when using the
```

Reply:
@lpaoletti I guess "control" would be a better word. Updated it.